### PR TITLE
Add first version of the complete SDK docs

### LINKED ISSUE #199: Document SDK code
Use the conventions to provide the documentation of the code for Iggy SDK crate.

### PR DESCRIPTION
**Pull request by spetz**
This closes #199.

### CODE REVIEW

**Review by hubcio — CHANGES_REQUESTED** [review]
Move "closes #199" to description

**Review by hubcio — APPROVED** [review]

File: iggy/src/identifier.rs [hunk 1]
```diff
@@ -8,21 +8,32 @@ use serde_with::serde_as;
...
+/// It consists of the following fields:
+/// - `kind`: the kind of the identifier.
+/// - `length`: the length of the identifier payload.
+/// - `value`: the binary value of the identifier payload.
 #[serde_as]
 #[derive(Debug, Serialize, Deserialize, PartialEq, Clone)]
 pub struct Identifier {
+    /// The kind of the identifier.
     pub kind: IdKind,
+    /// The length of the identifier payload.
```
Comment (numinnex) [inline iggy/src/identifier.rs:21]: Maybe it's worth including information about [min,max] length of Identifier ?

File: README.md [hunk 2]
```diff
@@ -14,6 +14,12 @@
 
 The goal of the project is to make a distributed streaming platform (running as a cluster), which will be able to scale horizontally and handle **millions of messages per second** (actually, **it's already very fast**, see the benchmarks below).
 
+Iggy provides the **very high throughput and performance** while utilizing a low amount of computing resources.
```
Comment (numinnex) [inline README.md]: I propose this instead: Iggy provides **exceptionally high throughput and performance** while utilizing minimal computing resources.

File: iggy/src/identifier.rs (hunk 1 above)
Comment (spetz) [reply iggy/src/identifier.rs:21]: Added max length.

File: README.md (hunk 2 above)
Comment (spetz) [reply README.md]: Updated :)

**Review by numinnex — APPROVED** [review]

**Pull request merged**